owtf/api/handlers/work.py

```diff
@@ -346,7 +361,9 @@ def post(self, work_id=None, action=None):
             if not target_list:
                 raise APIError(400, "Target list should not be empty")
             force_overwrite = str2bool(self.get_argument("force_overwrite", "False"))
```

Comment:
`force_overwrite` variable needs to be updated after the above change.

Reply:
Thank you for reviewing the changes.
I will look into it and make the required changes.

Reply:
correct, thanks @sharmamohit123

Reply:
@viyatb @sharmamohit123  Have made the required changes, Thank You.